Tools/AP_Periph/rc_out.cpp

```diff
@@ -154,6 +154,10 @@ void AP_Periph_FW::rcout_handle_safety_state(uint8_t safety_state)
 
 void AP_Periph_FW::rcout_update()
 {
+    if (!g.servo_enabled) {
```

Comment:
I'd prefer this to be completely automatic, based on if we have ever received a command, but there may be complications with that (eg. init at boot values, and possible outputs from within AP_Periph)
if that isn't viable then happy to accept this PR